win32ss/user/ntuser/winpos.c

```diff
@@ -3391,36 +3392,30 @@ NtUserGetWindowPlacement(HWND hWnd,
 {
    PWND Wnd;
    WINDOWPLACEMENT Safepl;
-   NTSTATUS Status;
    BOOL Ret = FALSE;
 
    TRACE("Enter NtUserGetWindowPlacement\n");
    UserEnterShared();
 
    if (!(Wnd = UserGetWindowObject(hWnd)))
-   {
       goto Exit; // Return FALSE
-   }
 
-   Status = MmCopyFromCaller(&Safepl, lpwndpl, sizeof(WINDOWPLACEMENT));
-   if (!NT_SUCCESS(Status))
+   _SEH2_TRY
    {
-      SetLastNtError(Status);
-      goto Exit; // Return FALSE
+      ProbeForRead(&lpwndpl->length, sizeof(lpwndpl->length), 1);
+      Safepl.length = lpwndpl->length; // Use length value
+      if (IntGetWindowPlacement(Wnd, &Safepl))
```

Comment:
This call doesn't need to be in the SEH2_TRY block (since you are working with captured data).
If it crashes (would be a bug on win32k side then), this would now be hidden because of it being in the SEH2_TRY block.